doc/source/cluster/index.rst

```diff
@@ -3,227 +3,33 @@
-* **Ray Nodes**: A Ray cluster consists of a **head node** and a set of **worker nodes**. The head node needs to be started first, and the worker nodes are given the address of the head node to form the cluster. The Ray cluster itself can also "auto-scale," meaning that it can interact with a Cloud Provider to request or release instances according to application workload.
-
-* **Ports**: Ray processes communicate via TCP ports. When starting a Ray cluster, either on prem or on the cloud, it is important to open the right ports so that Ray functions correctly. See :ref:`the Ray Ports documentation <ray-ports>` for more details.
-
-* **Ray Cluster Launcher**: The :ref:`Ray Cluster Launcher <ref-automatic-cluster>` is a simple tool that automatically provisions machines and launches a multi-node Ray cluster. You can use the cluster launcher on GCP, Amazon EC2, Azure, or even Kubernetes.
-
-Summary
--------
-
-Clusters are started with the :ref:`Ray Cluster Launcher <ref-automatic-cluster>` or :ref:`manually <manual-cluster>`.
-
-You can also create a Ray cluster using a standard cluster manager such as :ref:`Kubernetes <ray-k8s-deploy>`, :ref:`YARN <ray-yarn-deploy>`, or :ref:`SLURM <ray-slurm-deploy>`.
-
-After a cluster is started, you need to connect your program to the Ray cluster by starting a driver process on the same node as where you ran ``ray start``:
-
-.. tabs::
-  .. code-tab:: python
-
-    # This must
-    import ray
-    ray.init(address='auto')
-
-  .. group-tab:: java
-
-    .. code-block:: java
-
-      import io.ray.api.Ray;
-
-      public class MyRayApp {
-
-        public static void main(String[] args) {
-          Ray.init();
-          ...
-        }
-      }
-
-    .. code-block:: bash
-
-      java -classpath <classpath> \
-        -Dray.address=<address> \
-        <classname> <args>
-
-and then the rest of your script should be able to leverage Ray as a distributed framework!
-
-
-Using the cluster launcher
---------------------------
-
-The ``ray up`` command uses the :ref:`Ray Cluster Launcher <ref-automatic-cluster>` to start a cluster on the cloud, creating a designated "head node" and worker nodes. Any Python process that runs ``ray.init(address=...)`` on any of the cluster nodes will connect to the ray cluster.
-
-.. important:: Calling ``ray.init`` on your laptop will not work if using ``ray up``, since your laptop will not be the head node.
-
-Here is an example of using the Cluster Launcher on AWS:
-
-.. code-block:: shell
-
-    # First, run `pip install boto3` and `aws configure`
-    #
-    # Create or update the cluster. When the command finishes, it will print
-    # out the command that can be used to SSH into the cluster head node.
-    $ ray up ray/python/ray/autoscaler/aws/example-full.yaml
-
-You can monitor the Ray cluster status with ``ray monitor cluster.yaml`` and ssh into the head node with ``ray attach cluster.yaml``.
-
-.. _manual-cluster:
-
-Manual Ray Cluster Setup
+What is Distributed Ray?
```

Comment:
Hmm, why do we want to coin a new term called "Distributed Ray"? Why not just call it "Ray Cluster", so that it is not a new concept?

Otherwise, we should not capitalize "Distributed" everywhere in this documentation

Reply:
I changed it to "What is a Ray cluster?"